libraries/rush-lib/src/cli/scriptActions/PhasedScriptAction.ts

```diff
@@ -187,7 +186,8 @@ export class PhasedScriptAction extends BaseScriptAction<IPhasedCommandConfig> {
       rushConfiguration: this.rushConfiguration,
       phaseSelection: new Set(this._initialPhases),
       projectChangeAnalyzer: new ProjectChangeAnalyzer(this.rushConfiguration),
-      projectSelection
+      projectSelection,
+      changedProjects: projectSelection
```

Comment:
Seems like this needs to get updated.

Reply:
Calling this property `changedProjects` is confusing when reading through the downstream code. `projectSelection` makes more sense since it's the set of projects that are in the selected scope.

Reply:
`changedProjects` is the set of projects in the selected scope that are expressly known to be in a dirty state. `projectSelection` is all projects that are currently being tracked.

Reply:
Let's rename `changedProjects` to `projectsNeedingRerun` or `projectsInUnknownState`